input/pagecontent/2-5-EventNotification.md

```diff
@@ -146,3 +146,7 @@ During a normal shutdown of an application, it SHALL unsubscribe, and provide a
 * after the first time a Hub has distributed a [`syncerror`](3-2-1-syncerror.html) indicating that an application is not responsive, should the nonresponsive application be automatically unsubscribed (removed from the Hub's list of applications subscribed to the topic)?  This would avoid [`syncerror`](3-2-1-syncerror.html) events being sent after subsequent operations; however, it may conflict with the approach of [`syncerror`](3-2-1-syncerror.html) events generated by the Hub only being distributed to subscribers of the event which triggered the [`syncerror`](3-2-1-syncerror.html) event to be generated.
 * should [`syncerror`](3-2-1-syncerror.html) events generated by the Hub be distributed only to subscribers of the event which triggered the [`syncerror`](3-2-1-syncerror.html) event to be generated?  However, this could conflict automatically unsubscribe a non-responsive application after the initial [`syncerror`](3-2-1-syncerror.html) is generated and distributed.
 * should all FHIRcast requests trigger  [`syncerror`](3-2-1-syncerror.html) events to be generated by the Hub for an unresponsive application or only when a context change is requested ([FHIR Resource]-open or [FHIR Resource]-close)?
+
+### Hub Generated `open` Events
+
+If a Hub granted subscriptions to different sets of `hub.events` to different applications for the same session, the Hub is responsible for generation of implied open events. When distributing a received event, a hub SHALL ensure open events for the referenced resource types of the received event, are also sent to subscribers. Hubs SHOULD NOT generate and send duplicative events.

```

Comment:
First sentence:
* Should this be "If a Hub *grants*..."?
Second sentence:
* Should the "open" in "ensure open events" be formatted as `open`?
* I do not think the last comma is necessary (e.g., "...received event are also sent to subscribers."